stdlib/rdp/main.go

```diff
@@ -0,0 +1,41 @@
+package main
```

Comment:
I am guessing you are using `main.go` for running the test for the algorithm. 
This can be moved into go test file.

Reply:
This is an obsolete file and was intended to be part of this commit. Will remove them